src/hooks/use-run-tsx/index.tsx

```diff
@@ -21,10 +19,12 @@ type RunTsxResult = {
 
 export const useRunTsx = ({
   code,
+  userImports,
   type,
   isStreaming = false,
 }: {
   code?: string
+  userImports?: Record<string, string>
```

Comment:
If i understand your change correctly, userImports should map to an object now right?

```suggestion
  userImports?: Record<string, object>
```